cms/serializers_test.py

```diff
@@ -147,7 +147,7 @@ def test_serialize_course_page_with_flex_price_form_as_program_child(
     program = ProgramFactory()
     program_page = ProgramPageFactory(program=program)
     FlexiblePricingFormFactory(parent=program_page)
-    course = CourseFactory(program=program)
+    course = CourseFactory(program=program, page=None)
```

Comment:
Any specific reason we want the page to be sent as `None`?

Reply:
The `page` is added to CourseFactory as discussed earlier for testing the Enrollment Serializer. But it caused the failure of other tests because previously the `page` attribute (which is referred to CoursePage) was None by default and created after CoursePageFactory.create(). Now after adding `page` attribute in CourseFactory the `page` attribute is created as a CoursePage as default. This caused the error `Course page with this Course already exists.` when we call CoursePageFactory. So passing the page None allow to call CoursePageFactory